### PR TITLE
Fix mistakes + Add some updates [Fandogh PaaS plugin]

### PR DESCRIPTION
**Pull request by ohmydevops**
- add custom registry url
- fix parameters

### CODE REVIEW

**Comment by amirbagh75** [comment]
please check this PR 
@tboerger

**Comment by amirbagh75** [comment]
please check this PR
@tboerger

**Comment by amirbagh75** [comment]
please check this PR
@tboerger

File: content/amirbagh75/drone-fandogh/index.md [hunk 1]
```diff
@@ -34,16 +34,55 @@ Example configuration using secrets:
...
+  settings:
+    username: 
+      from_secret: fandogh_username
+    password: 
+      from_secret: fandogh_password
+    version: ${DRONE_COMMIT}
+    registry: myregistry.com:5000
+    manifest_path: ./fandogh.yml
+```
+Example fandogh manifest (with custom registry url):
```
Comment (tboerger) [inline content/amirbagh75/drone-fandogh/index.md:51]: Add an empty line before this

File: content/amirbagh75/drone-fandogh/index.md [hunk 2]
```diff
@@ -34,16 +34,55 @@ Example configuration using secrets:
...
+  replicas: 1
+  port_mapping:
+  - port: 50052
+    target_port: 50052
+    protocol: tcp
+  env:
+  - name: GO_ENV
+    value: "production"
+```
+
```
Comment (tboerger) [inline content/amirbagh75/drone-fandogh/index.md:68]: Only one empty line, please.

**Comment by amirbagh75** [comment]
please check this PR
@tboerger

**Comment by ohmydevops** [comment]
Okay, I'll do it

**Pull request closed without merging**